pages/events/index.mdx

```diff
@@ -27,11 +29,15 @@ Different events will provide different data - some even ask for data back, such
 
 In the case of our example, the event requires an `ActionResult`, the javadoc states the following `ActionResult` types can be used:
 
-|Type|Affect|
-|----|---------------|
-|`ActionResult.SUCCESS`|Cancels all other listeners, cancels the attack as well.|
-|`ActionResult.PASS`|Do not do anything else.|
-|`ActionResult.FAIL`|Cancel all other listeners, but don't cancel the attack.|
+| Type                   | Affect                                                               |
+|------------------------|----------------------------------------------------------------------|
+| `ActionResult.SUCCESS` | Cancels further processing and sends a packet to the server.         |
+| `ActionResult.PASS`    | Falls back to further processing.                                    |
+| `ActionResult.FAIL`    | Cancels further processing and does not send a packet to the server. |
+
+<Callout type="info">
+    When `ActionResult.FAIL` is returned, the Server will not be informed about the Event, and as a result, the Server will not process it.
```

Comment:
Dont capitalize server, other than that, all good!

Reply:
done